cli/install/install.go

```diff
@@ -337,20 +345,19 @@ func downloadRepo(repoLink string, tag string, dst string,
 	logFile *os.File, verbose bool) error {
 	var err error
 	if tag == "master" {
-		err = util.ExecuteCommand("git", verbose, logFile, dst, "clone",
-			"-j", "18", repoLink,
```

Comment:
Why `-j n` flag was removed?

Reply:
This option is not supported in older git versions, which makes `tt install` non-working on some Linux distributions.